packages/daemon/src/identity/IdentitiesRepository.js

```diff
@@ -164,6 +186,27 @@ const createOwnGroup = (params: OwnIdentitiesRepositoryGroupParams = {}) => ({
   users: params.users == null ? {} : params.users,
 })
 
+const contactsFromJSON = mapObject(Contact.fromJSON)
+const contactsToJSON = mapObject(Contact.toJSON)
+
+const toContacts = (params: ContactsRepositorySerialized = {}) => {
+  const contacts = {}
+  Object.keys(params).forEach(ownID => {
+    // $FlowFixMe: mapper type
+    contacts[ownID] = contactsFromJSON(params[ownID])
+  })
+  return contacts
```

Comment:
Not an issue but rather than using `forEach()` and an external variable, it's usually recommended to use `reduce()` for this kind of case so variable is internal, ex:
```js
  return Object.keys(params).reduce((acc, ownID) => {
    // $FlowFixMe: mapper type
    acc[ownID] = contactsFromJSON(params[ownID])
    return acc
  }, {})
```